src/test/java/tests/FilterEvalTests.java

```diff
@@ -433,6 +434,38 @@ public void created() {
     }
 
     @Test
+    public void updatedByOther(){
+        LocalDateTime now = LocalDateTime.now();
```

Comment:
We can probably merge these two tests, since they're testing very similar things. You don't have to test ranges, since they are covered elsewhere. Something more important is which qualifiers match in which scenarios: given an issue, there are 4 possibilities for whether the two qualifiers match, so that would be more useful to focus on.

```java
TurboIssue issue = ...;
assertEquals(false, matches("updatedByOthers:24", issue));
assertEquals(false, matches("updatedBySelf:24", issue));
// repeat for (true, true), (false, true), (true, false)
```
There's also the cases where issues can be updated by **both** others and self, or **none**. A test to demonstrate which takes precedence would be useful.